business/istio_validations.go

```diff
@@ -47,33 +50,35 @@ func (in *IstioValidationsService) GetServiceValidations(namespace, service stri
 
 func (in *IstioValidationsService) GetNamespaceValidations(namespace string) (models.NamespaceValidations, error) {
 	// Get all the Istio objects from a Namespace
+	// TODO This isn't getting all of them, only blocking processing so we can fetch rest later. Needs refactoring
```

Comment:
I agree.
It's clear we can't continue doing individual operations per element, as we need at least info from the entire namespace to validate an element.